modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -643,6 +657,25 @@ private void cleanup(CacheConfiguration cfg, @Nullable Object rsrc, boolean near
             this::partStatesViewSupplier,
             Function.identity()
         );
+
+        ctx.systemView().registerView(
+            CACHE_GRP_IO_VIEW,
+            CACHE_GRP_IO_VIEW_DESC,
+            new CacheGroupIoViewWalker(),
+            () -> F.view(cacheGrps.values(), grp -> !grp.systemCache()),
+            grpCtx -> {
+                MetricRegistry mreg = ctx.metric().registry(metricName(IoStatisticsType.CACHE_GROUP.metricGroupName(),
+                    grpCtx.cacheOrGroupName()));
+
+                boolean hasMetrics = mreg.findMetric("grpId") != null;
+
+                return new CacheGroupIoView(
+                    grpCtx,
+                    hasMetrics ? mreg.<LongMetric>findMetric(PHYSICAL_READS).value() : 0,
```

Comment:
Let's move this logic inside `CacheGroupIoView` and have simpler constructor `return new CacheGroupIoView(grpCtx, mreg)`